modules/disk.sh

```diff
@@ -96,7 +96,7 @@ disk_align_sectors() {
 # $2 - destination file
 # $3 - offset in number of 512 sectors
 disk_write_at_offset() {
-  run_and_log_cmd "dd if=${1} of=${2} seek=${3} conv=sparse,notrunc status=none"
+  run_and_log_cmd "dd if=${1} of=${2} seek=${3} conv=notrunc status=none"
```

Comment:
I do not understand how this would cause problems? Can you give an example?

Reply:
I didn't understand it either but it definitely did make a difference.  Last week when I tested it. ;-)
Now for some reason it doesn't seem to be needed. Who knows.